src/data/scripts/internet/main.py

```diff
@@ -12,57 +12,40 @@
     classify_spam,
     sample_text_back,
 )
+from openthaigpt_pretraining_data.core.processing_config import load_config
+from openthaigpt_pretraining_data.core.metadata import (
+    create_info_file,
+    create_metadata_file,
+)
 import numpy as np
 import scipy
 import json
 import os
-import hydra
 import zstandard as zstd
+import argparse
 
-NUM_PROC = 128
-
-do_perplexity = batch_size = sampled_back_ratio = None
-output_dir = scratch_location = version = None
-source = input_path = input_version = note = None
-
-
-@hydra.main(config_path=".")
-def load_config(cfg):
-
-    global do_perplexity, batch_size, sampled_back_ratio, input_version
-    global output_dir, scratch_location, source, input_path, version, note
-
-    cfg = cfg.config
-
-    do_perplexity = cfg.processing_parameters.do_perplexity
-    batch_size = cfg.processing_parameters.batch_size
-    sampled_back_ratio = cfg.processing_parameters.sampled_back_ratio
-
-    output_dir = cfg.output.path
-    scratch_location = cfg.output.scratch_path if "scratch_path" in cfg.output else None
-
-    input_based_path = cfg.input_dataset.path
-
-    info = json.load(open(f"{input_based_path}/info.json", "r"))
```

Comment:
Why we remove logic to get info from input’s info.json

Reply:
Just move it to `core/processing_config.py` krub